types/storybook-readme/src/doc.d.ts

```diff
@@ -0,0 +1 @@
+export type Doc = (readme: string) => any;
```

Comment:
I'm not sure this deserves its own module. It doesn't look like `storybook-readme` recommends people to import directly from its `src` directory. I think these type-only modules could just be moved to be part of `index.d.ts`.